authors/apps/authentication/views.py

```diff
@@ -88,3 +115,28 @@ def update(self, request, *args, **kwargs):
         serializer.save()
 
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class AccountActivationAPIView(APIView):
+    """
+    Return the registration activation link.
+    """
+    permission_classes = (AllowAny, )
+    serializer_class = UserSerializer
+
+    def get(self, request, uid):
+        try:
+            username = urlsafe_base64_decode(uid).decode('utf-8')
+            user = User.objects.filter(username=username).first()
```

Comment:
@mwinel Confirm this throws the exception(User.DoesNotExist) you expect it to throw.